roles/fio-distributed/templates/server_vm.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+    app: fio-benchmark-{{ trunc_uuid }}
+  name: 'fio-server-{{ item }}-benchmark-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+spec:
+{% if pin_server %}
+  nodeSelector:
+    kubernetes.io/hostname: '{{ pin_server }}'
+{% endif %}
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+{% if fiod.storageclass is defined %}
+      - name: test-volume
+        # This makes it a disk
+        disk: {}
+        serial: CVLY623300HK250D
+{% endif %}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
```

Comment:
Should this be static? If there is a issue, we literally have to change the code in the operator, it might be better to allow the user to change this.